test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/survival/KieServerWithWorkbenchSurvivalIntegrationTest.java

```diff
@@ -148,10 +138,15 @@ private void sendRESTCallsToUnavailableKieServer() {
                                 deploymentScenario.getKieServerDeployment().getPassword()))
                 .build();
 
-        WebTarget target = httpKieServerClient.target(deploymentScenario.getKieServerDeployment().getUrl().toString() + GET_CONTAINERS_REST_ENDPOINT);
-        Response response = target.request().get();
-        response.close();
-        assertThat(response.getStatus()).isEqualTo(Response.Status.SERVICE_UNAVAILABLE.getStatusCode());
+        try {
+            URL url = new URL(deploymentScenario.getKieServerDeployment().getUrl(), KIE_CONTAINER_REQUEST_URL);
+            WebTarget target = httpKieServerClient.target(url.toString());
+            Response response = target.request().get();
```

Comment:
This is a **much bigger task** than could ever be accomplished in this PR, but much of our REST-specific code could be completely eliminated if we just used the RESTEasy client proxy:

https://docs.jboss.org/resteasy/docs/3.0-beta-3/userguide/html/RESTEasy_Client_Framework.html#d4e2049

We could directly reuse the APIs for the services, without having to hard-code the URLs, HTTP methods etc.